modules/mediawiki/manifests/rsync.pp

```diff
@@ -0,0 +1,30 @@
+# === Class mediawiki::rsync
+#
+# MediaWiki remote sync configuration
+class mediawiki::rsync {
+    users::user { 'www-data':
```

Comment:
Should be before nginx as it in theory sets uid/gid

Reply:
Done